crates/turborepo-lib/src/daemon/server.rs

```diff
@@ -285,9 +291,15 @@ impl<T: Watcher + Send + 'static> proto::turbod_server::Turbod for DaemonServer<
             )
             .await;
 
+        let mut timeSaved = 0;
+        if let Some(value) = self.times_saved.get(&inner.hash.clone()) {
+            timeSaved = *value;
+        }
```

Comment:
I can't add a code suggestion, but above `let changed = self` we should add a
`let hash = Arc::new(inner.hash);` and change the first argument from `&Arc::new(inner.hash);` to `&hash`.

Breaking down the other changes:
 - `copied`: This is a helper method that changes an `Option<&T>` to an `Option<T>` if `T` implements `Copy`
 - `unwrap_or_default`: converts an `Option<T>` to `T`, if the value is present it will be used, but if `None` than we use `T`'s default

Questions I have here: do we want to paper over missing a time saved? 
```suggestion
        let timeSaved = {
            let times_saved = self.times_saved.lock().expect("times saved lock poisoned");
            times_saved.get(hash.as_str()).copied().unwrap_or_default()
        };
```